third_party/move/tools/move-formatter/tests/formatter/tuple/input5.fmt.move

```diff
@@ -0,0 +1,31 @@
+address 0x42 {
+module example {
+    /**
+    * This function doesn't return anything. It's just for demonstration purposes. */
```

Comment:
This line looks a little bit strange to me.

Reply:
I can not get your point.